extension/package.json

```diff
@@ -33,14 +33,15 @@
   "contributes": {
     "commands": [
       {
-        "command": "component-viewer",
-        "title": "XCViewer: View cxml file"
+        "command": "xcomponent.preview",
+        "title": "XComponent: preview component",
```

Comment:
it is the name of our extension command.
It's better to preview our commands with the name of our extension.
It seems to be a best practice...